test/hotspot/jtreg/vmTestbase/nsk/jvmti/scenarios/jni_interception/JI06/ji06t001/ji06t001.cpp

```diff
@@ -36,28 +36,27 @@
 
 extern "C" {
 
-#define PASSED  0
-#define STATUS_FAILED  2
+#define PASSED 0
 
 #define TRIES 30
 #define MAX_THREADS 5
+#define STACK_SIZE 1024*1024
 
 // Helper for thread detach and terminate
 #define THREAD_return(status) \
   do { \
       int res = vm->DetachCurrentThread(); \
-      if (res != 0) \
-          NSK_COMPLAIN1("TEST WARNING: DetachCurrentThread() returns: %d\n", res); \
-      else \
+      if (res == 0) \
           NSK_DISPLAY0("Detaching thread ...\n"); \
-      return status; \
-  } while (0)
-
+          return 0; \
+      exit(-1); \
```

Comment:
The indent of the  `return statement` is wrong. It's at the same level as the `exit()`, which means the `exit()` is never executed.

This code now ignores the `status` argument, so it doesn't seem to be correct.

I'm not sure why any of the changes here were necessary.

Reply:
I have applied changes to native_thread.cpp instead of changing in every executables. Please review the change. It will be working fine on windows as well.